tests/test_data/demo.sql

```diff
@@ -1786,9 +1786,12 @@ INSERT INTO public.action_translations VALUES (14, -99, 14, 'choices translation
 INSERT INTO public.action_type_translations VALUES (1, -99, -99, 'Sample Creation', 'These Actions represent processes which create a sample.', 'Sample', 'Samples', 'View Samples', 'Create Sample');
 INSERT INTO public.action_type_translations VALUES (2, -99, -98, 'Measurement', 'These Actions represent processes which perform a measurement.', 'Measurement', 'Measurements', 'View Measurements', 'Perform Measurement');
 INSERT INTO public.action_type_translations VALUES (3, -99, -97, 'Simulation', 'These Actions represent processes which run a simulation.', 'Simulation', 'Simulations', 'View Simulations', 'Run Simulation');
+INSERT INTO public.action_type_translations VALUES (3, -99, -96, 'Template', 'These action type creates the possibility to create templates for creating other actions.', '', '', '', '');
```

Comment:
```suggestion
```
(1/3) This sql dump represents an older state of SampleDB and is used to test the migrations.
As the Template action type is added in a migration it should not be created here.
This leads to a failing migration test.
The name `demo.sql` might be a little misleading here..